tests/unit/setup-context-test.js

```diff
@@ -849,10 +849,7 @@ module('setupContext', function (hooks) {
 
     hooks.beforeEach(function () {
       const AppConfig = assign({ autoboot: false }, config.APP);
-      // .extend() because initializers are stored in the constructor, and we
-      // don't want to pollute other tests using an application created from the
-      // same constructor.
-      isolatedApp = App.extend({}).create(AppConfig);
```

Comment:
The thing this comment is describing is absolutely an issue (and why you had to add that `delete` in the after each), we can't share the same parent as `App` here. 

Can you bring back the comment, and assuming the goal was to get rid the manual `.extend()` call we can change to:

```js
isolatedApp = (class extends App {}).create(AppConfig);
```

This does the same thing as the `.extend` did and doesn't require the `delete` you added to the `afterEach`

Reply:
Thanks and addressed!